src/Application/Components/Store/Store.jsx

```diff
@@ -1,29 +1,28 @@
 import React from 'react';
 import StoreItem from './StoreItem';
-import _ from 'underscore';
 
 const Store = React.createClass({
   propTypes: {
     assets: React.PropTypes.array.isRequired,
-    createOnClick: React.PropTypes.func.isRequired,
+    onItemClick: React.PropTypes.func.isRequired,
   },
 
   render() {
-    const buyFxn = this.props.createOnClick;
-    const unlockedAssets = _.where(this.props.assets, { unlocked: true });
-
     return (
         <div id="store-zone">
           <h3>Broker</h3>
           <div id="item-menu">
-            {unlockedAssets.map(function (asset) {
+            {this.props.assets.map((asset, index) => {
+              const onClick = () => {
+                this.props.onItemClick(index);
+              };
```

Comment:
we tend to use `_.partial(this.props.onItemClick, index)` but i'm not sure there's any actual difference